templates/workspaces/base/terraform/outputs.tf

```diff
@@ -1,3 +1,19 @@
 output "workspace_resource_name_suffix" {
   value = local.workspace_resource_name_suffix
-}
\ No newline at end of file
+}
+
+output "WorkspaceOwner" {
```

Comment:
Can we be more explicit? WorkspaceOwnerAppRoleId?

Reply:
Sure - this is because the json stored in cosmos is just "WorkspaceOwner". I'm making a breaking change here anyway, so I'll be more explicit.